### DISCUSSION TITLE
How to load the Nemesis mesh group ?

### DISCUSSION THREAD

**Discussion by mangerij** [idaholab/moose#17137] [Q&A General] [answered]
Say I have outputs from a single calculation

out_PTOfilm_e25_T298K_E0_E0.e.**N**.**M**

where **N** is number of total processors and **M** is the processor index. I want to load the group of these files as my mesh for a new calculation (same partitioning). Is this possible?

Along with `file = out_PTOfilm_e25_T298K_E0_E0.e`, I tried to add `nemesis=true` and `skip_partition= true` in the [Mesh] block and that does nothing.

I also checked the _test/tests/mesh/nemesis_test.i_ test and I have the same syntax to load the files.

The weird thing is that the error message says that it sees the files but for example,

*** ERROR ***
Error opening ExodusII mesh file: /lcrc/project/Meso/projects/ferret/out_PTOfilm_e25_T298K_E0_E46.e.256.060

thanks
John

**Comment by PM-Hu** [comment]
Hi
I'm new to MOOSE, and also meet that error when I was intending to use the Exodus file generated by Trelis in Mesh block.
The `chmod` command may solve that error to change the ExodusII mesh file's [out_PTOfilm_e25_T298K_E0_E46.e.256.060] mode.

hope that helps you
Hu

**Comment by GiudGiud** [comment, marked as answer]
Is this solved?

**Reply by GiudGiud** [reply]
I checked with the example moose/test/tests/meshgenerators/distributed_rectilinear/dmg_displaced_mesh there, loading modifying it to load the nemesis mesh group and it worked fine. 

Closing this

**Reply by Traiwit** [reply]
im having hard time loading the variable data back to MOOSE via nemesis mesh

in exodus i usually do:

```
  [mesh_G01Hv1]
    type = FileMeshGenerator
    file = 'XXXX.e'
    allow_renumbering = false
    use_for_exodus_restart = true  ## for restart_mode
  []
```

then 

```
[Variables]
  [porepressure_L1]
    type = MooseVariable
    order = FIRST
    family = LAGRANGE
    initial_from_file_var = porepressure_L1  ## for restart_mode
    initial_from_file_timestep = LATEST      ## for restart_mode   
  []
[]
```

but this wont work with nemesis

even if i do

```
  # file = '/home/moose/Downloads/LKABKIR2025081/2_RUN_MOOSE/R09HR01/outputNemesis/R11HR01/LKABKIR2025081_R11HR01_F004.e'
  # nemesis = true
  # skip_partitioning = true
  # use_for_exodus_restart = true 
  
  under [Mesh]
```
  
  i will have problem with loading variable data and assigning data from `PropertyReadFile`
  
  is there a way to work around?
  
  `SolutionUserObject `- wont work as well, as it only takes .e
  
  Regards,
Traiwit

**Reply by Traiwit** [reply]
or is there a way to convert nemesis back to single .e and i can restart from there?

**Reply by GiudGiud** [reply]
Hello

Exodus restart does not support nemesis I think

However I have a PR open to use nemesis for restarts via solution user objects if you want to try that out

**Reply by Traiwit** [reply]
i see, thank you @GiudGiud 

I went with the harder route: 

-> convert nemesis files into .e via external py script

-> then restart simulation with .e and back to nemesis output again

seems to work for now but painful

**Reply by GiudGiud** [reply]
Yeah that should work. 
You just lose the distrivuted side of nemesis which the solutionUO technique should not (its the intent of that PR)
